### DISCUSSION TITLE
What is Asset Tag?

### DISCUSSION THREAD

**Discussion by iglov** [netbox-community/netbox#17684] [General]
```
Asset Tag

A unique, locally-administered label used to identify hardware resources.
```
(c) https://netboxlabs.com/docs/netbox/en/stable/models/dcim/device/#asset-tag

Is this some kind of physical mark on the device to identify it in the rack or something else?
<img width="313" alt="image" src="https://github.com/user-attachments/assets/c89ddb39-fab7-4202-ac6e-3486db220417">
4 ex.: Is it an Asset Tag?

**Comment by jeremystretch** [comment]
Conceptually, asset tags are unique labels issued by an organization to individual pieces of hardware for tracking and accounting purposes. This might extend beyond servers and network infrastructure to peripherals like desktop monitors and other equipment as well.

Not all organizations use asset tags, but NetBox includes an asset tag field on several models for those that do.

**Reply by iglov** [reply]
hey @jeremystretch ! Thx 4 reply!
Well, i see... but everyone who has racks has some kind of "physical tags" on the devices in those racks, like the one in the image above, to tell remote hands to identify those devices. So-o-o-o, where i can put those "physical tags" in NetBox? Should i create custom field or maybe it is a good idea to create a feature request to create a new default field (like "Asset tag") in NetBox?

**Reply by candlerb** [reply]
For the "SB01" shown, it sounds to me like it should be the device name.

If you have a label that's not the device name, and is not the asset tag, then indeed a custom field may be appropriate. But it's usually a good plan to label devices with the name that they are known as. That is IMO the whole point of a name.

**Reply by DanSheps** [reply]
> Well, i see... but everyone who has racks has some kind of "physical tags" on the devices in those racks, like the one in the image above, to tell remote hands to identify those devices. So-o-o-o, where i can put those "physical tags" in NetBox? Should i create custom field or maybe it is a good idea to create a feature request to create a new default field (like "Asset tag") in NetBox?

That is likely the name.  An asset tag is normally a tag generated by your finance or shipping/receiving department.  For example, where I work anything over $1000 (I think) gets an asset tag.  Buy a $5000 chair?  Asset tag.  A filing cabinet > $1000?  Asset tag.  Laptop > $1000?  Asset tag.  CPU for insertion into a Cisco UCS blade?  Asset tag (Yes, we have bought CPU's over $1000 and almost had to clean it before installing because shipping wanted to slap a sticker on it.)

**Reply by iglov** [reply]
Ok, if I use the name field for this, where should I put the hostname? The hostname != this "physical tag".

**Reply by candlerb** [reply]
Depends what you mean by "hostname". Can you give an example? e.g. what is SB01's "hostname"?

If you mean the DNS name, and your DNS names are not constructed as name + a fixed domain, then you can use the dns_name field on the IP address object.  Or you can use the DNS name as the Netbox device name, and then the printed label is just the first part of the DNS name.

That might not work well if you have lots of different "sb01" devices under different domains, but then again, that could also lead to confusion when people talk informally about "sb01" and it could mean different things. In the end, you have to choose a naming scheme that works well for your environment.

**Reply by DanSheps** [reply]
> Ok, if I use the name field for this, where should I put the hostname? The hostname != this "physical tag".

IMO, you should not have device where the physical "name" and the hostname vary (beyond FQDN since you wouldn't do the full FQDN on the label, but you might do SB01.nyc)

Asset tag must be unique globally, so if you have 2 SB01's you are cooked.

**Reply by iglov** [reply]
ehh, i see, its better to create a custom field :facepalm:
thx guyz!

**Reply by DanSheps** [reply]
You could also put additional details in the description field.
